apps/workflows/api.py

```diff
@@ -128,6 +126,88 @@ def post(self, request, flaw_id):
             return Response({"errors": str(e)}, status=status.HTTP_409_CONFLICT)
 
 
+class RevertWorkflow(RudimentaryUserPathLoggingMixin, APIView):
```

Comment:
Should we rename existing API views here to match these newly introduced? :thinking: 
`promote` -> `PromoteWorkflow`
`reject` -> `RejectWorkflow`